### PR TITLE
Nodejs-week3

### PR DESCRIPTION
**Pull request by viedamayanti**

### CODE REVIEW

**Review by Yuliia-Balandiuk — COMMENTED** [review]
Hello Vin. Perfect job!

File: nodejs/week3/app.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  //The fix version (Solution)
+  if ('sort' in req.query) {
+    // @ts-ignore
+    const sorting = req.query.sort.toString().split(' ');
+    if (sorting.length > 0) {
+      query = query.orderBy(sorting[0]);
+    }
+  }
+  console.log('SQL', query.toSQL().sql);
+
```
Comment (Yuliia-Balandiuk) [inline nodejs/week3/app.js:47]: You are really strong in nod and I will be grateful for your help.

File: nodejs/week3/app.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  //The fix version (Solution)
+  if ('sort' in req.query) {
+    // @ts-ignore
+    const sorting = req.query.sort.toString().split(' ');
+    if (sorting.length > 0) {
+      query = query.orderBy(sorting[0]);
+    }
+  }
+  console.log('SQL', query.toSQL().sql);
```
Comment (Divyajg) [inline nodejs/week3/app.js:46]: Good idea to cross-check the query. 👍

**Review by Divyajg — APPROVED** [review]
Great work ⭐
Keep coding. 👍

**Comment by viedamayanti** [comment]
Thank you so much for feedback everyone, I appreciate it 🙏🌸

**Pull request merged**